frontend/src/MakerApp.tsx

```diff
@@ -187,18 +185,6 @@ export default function App() {
     );
 }
 
-function priceToDisplay(hasEnteredPrice: boolean, orderPrice: string, priceInfo: PriceInfo | null) {
-    if (!priceInfo) {
-        return format("0");
-    }
-
-    if (!hasEnteredPrice) {
-        return format((priceInfo.ask * SPREAD).toString());
```

Comment:
If you delete this, we are no longer calculating a default price using the spread.

Reply:
is there a way of triggering the "refresh" button when the `priceInfo` becomes available? That would solve the actual problem in a much nicer way IMHO.